pkgvars

```diff
@@ -1,8 +1,7 @@
 rpm_pkgname=pg-auto-failover
 deb_pkgname=auto-failover
+pkgname=pg-auto-failover
```

Comment:
I needed to add this since I'm also using pgkname to check in the code whethere it is hll or not 
https://github.com/citusdata/tools/blob/3f421528419e8d475dc579cab80022487f0b74f4/packaging_automation/citus_package.py#L303